chords_ble.py

```
@@ -0,0 +1,207 @@
+            
+            if current_unrolled != self.prev_unrolled_counter + 1:
+                missing = current_unrolled - (self.prev_unrolled_counter + 1)
+                print(f"Missing {missing} sample(s)")
+                self.total_missing_samples += missing
+            
+            self.prev_unrolled_counter = current_unrolled
+                   
+        if self.start_time is None:
+            self.start_time = time.time()
+        
+        channels = [
```

Ruff (0.8.2)

Local variable `channels` is assigned to but never used

Remove assignment to unused variable `channels`

(F841)